client/rest-high-level/src/test/java/org/elasticsearch/client/core/TermVectorsResponseTests.java

```diff
@@ -200,4 +207,14 @@ protected TermVectorsResponse createTestInstance() {
         return term;
     }
 
+    // generate a new string different from the supplied usedStrings
+    private String produceUniqueString(List<String> usedStrings) {
```

Comment:
@cbuescher  Thanks for the suggestion, I will remember about this next time.

Reply:
On a related note, there is a helper in `ESTestCase` that might be usable here: randomValueOtherThanMany(). I think in this method could be used here instead of `produceUniqueString`, somehow like `String fieldName = randomValueOtherThanMany(t -> usedStrings.contains(t), () -> randomAlphaOfLength(7))` (I might have the predicate the wrong way around). Just as a suggestion as well.